packages/pytest/meta.yaml

```diff
@@ -16,4 +19,10 @@ requirements:
     - iniconfig
 test:
   imports:
+    - _pytest
```

Comment:
I don't think we need to test it, it's an internal feature of pytest. All we care that pytest import works (and it likely wouldn't if it can't import _pytest)